src/AI/AILibrary.java

```diff
@@ -0,0 +1,44 @@
+package AI;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.Commander;
+import Engine.GameInstance;
+
+public class AILibrary
+{  
+  private static ArrayList<AIMaker> AIList = null;
+
+  public static ArrayList<AIMaker> getAIList()
+  {
+    if( null == AIList )
+    {
+      buildCommanderList();
+    }
+    return AIList;
+  }
+
+  private static void buildCommanderList()
```

Comment:
`buildCommanderList`?

Reply:
AIs are in fact commanders. You have *absolutely* no evidence that this code was copied from somewhere else.
At all.
:P